src/server/models/list.ts

```diff
@@ -62,7 +62,7 @@ export async function findListById(
 }
 
 export async function findListByCountryAndType(
-  country: CountryName,
+  country: CountryName | string,
```

Comment:
is CountryName not typeof string?

Reply:
I think it's a good idea to keep mapped types (eg countriesList[number]["value"]) because it's more descriptive of where the string has come from/what it relates to. In getListIdForCountryAndType you can do "as countriesList[number]["value"]"

Reply:
Thanks for the pointer, Jen.  I've made the requisite suggestions to ensure the type is used at all times.